intellij/test/junit/saros/intellij/filesystem/IntelliJReferencePointManagerTest.java

```diff
@@ -0,0 +1,107 @@
+package saros.intellij.filesystem;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.roots.ModuleRootManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import org.easymock.EasyMock;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import saros.filesystem.IPath;
+import saros.filesystem.IReferencePoint;
+
+public class IntelliJReferencePointManagerTest {
+
+  IReferencePoint referencePoint;
+  IPath fileReferencePointRelativePath;
+  IPath folderReferencePointRelativePath;
+  IPath projectFullPath;
+  IntelliJReferencePointManager intelliJReferencePointManager;
+  VirtualFile file;
+  VirtualFile folder;
+  VirtualFile moduleRoot;
+  String moduleRootPath;
+  ModuleRootManager manager;
+
+  @Before
+  public void setup() {
+    // moduleRootPath = "path/to/foo";
+    moduleRootPath = "";
```

Comment:
`moduleRootPath` seems to be an empty string for all tests. Why don't you introduce a `static final MODULE_ROOT_PATH = "";` instead of setting the value to `""` before each test.

Reply:
I rewrote the tests